.gitignore

```diff
@@ -0,0 +1,36 @@
+_site
```

Comment:
NOTE: added some Jekyll-specific files/folders to ignore (namely the compiled/built site)